lightyear/src/client/input.rs

```diff
@@ -140,43 +141,52 @@ impl<P: Protocol> Plugin for InputPlugin<P> {
                 .chain(),
         );
         app.configure_sets(FixedPostUpdate, InputSystemSet::ClearInputEvent);
-        if !app.world.resource::<ClientConfig>().is_unified() {
-            app.configure_sets(
-                PostUpdate,
-                (
-                    // handle tick events from sync before sending the message
-                    InputSystemSet::ReceiveTickEvents.after(SyncSet).run_if(
-                        // there are no tick events in unified mode
-                        client_is_synced::<P>,
-                    ),
-                    // we send inputs only every send_interval
-                    InputSystemSet::SendInputMessage
-                        .in_set(InternalMainSet::<ClientMarker>::Send)
-                        .run_if(
-                            // no need to send input messages via io if we are in unified mode
-                            client_is_synced::<P>,
-                        ),
-                    InternalMainSet::<ClientMarker>::SendPackets,
-                )
-                    .chain(),
-            );
+        // SYSTEMS
+        app.add_systems(
+            FixedPostUpdate,
+            clear_input_events::<P::Input>.in_set(InputSystemSet::ClearInputEvent),
+        );
+
+        if app.world.resource::<ClientConfig>().shared.mode == Mode::HostServer {
```

Comment:
I think it might be better for this to be controlled via run conditions so it can be toggled at runtime

Reply:
Yep, definitely, having things more run-time configurable is the next thing I will work on. I'll just keep things like this for now since everything is very static, and I'll work on the runtime configurability in a separate PR